src/settings.h

```diff
@@ -38,37 +38,91 @@
 #include "server.h"
 #include "util.h"
 
+/* Defines ********************************************************************/
+
+// In the new inifile format, both server and client settings can be stored in the same inifile.
```

Comment:
I wouldn't call it "new" format. Better give a Jamulus version here.

Reply:
> I wouldn't call it "new" format. Better give a Jamulus version here.

Why isn't it "new" ??? Did you read the next comment lines too?? If we save in the "new" format it can no longer be read by "old" versions of Jamulus...
And I can't give a Jamulus version yet, since I don't know when it will be implemented...